Kite SDK/PSPrintSDK/OLKitePrintSDK.m

```diff
@@ -232,10 +232,18 @@ + (NSString *)instagramClientID{
 }
 #endif
 
-+ (void)fetchRemotePlist{
++ (void)fetchRemotePlistsWithCompletionHandler:(void(^)())handler{
+    [OLKitePrintSDK fetchRemotePlistWithURL:@"https://sdk-static.s3.amazonaws.com/kite-ios-remote.plist" completionHandler:^(NSError *error){
+        [OLKitePrintSDK fetchRemotePlistWithURL:[NSString stringWithFormat:@"https://sdk-static.s3.amazonaws.com/kite-ios-remote-%@.plist", [OLKitePrintSDK apiKey]] completionHandler:^(NSError *error2){
```

Comment:
hah really elegant approach *but* I'd actually much prefer it if we hit the API key one first, and then only if it fails go fetch the next one. The reason is that for big partners it'll only be a single request delaying loading rather than a guaranteed two which has two RTTs vs one.